test/src/test/java/org/corfudb/integration/AbstractIT.java

```diff
@@ -735,9 +735,12 @@ public String getOptionsString() {
                 command.append(" --max-snapshot-entries-applied=").append(maxSnapshotEntriesApplied);
             }
 
+            if (corfuServerConnectionPort != 0) {
```

Comment:
Based on the fact that this has a default value, I don't think we need the intermediate corfuServerConnectionPort, you can just add to the command.

Reply:
If we add to the command without checking, port number will be taken as 0, right?  The default value will not be applied.  Am I missing something?

Reply:
It's only used to append to the command. But `--corfu-port-for-lr` will always have a value, cause if not set, it will be explicitly set to its default 9000, so no need to check.

Reply:
For tests which use multiple nodes in a cluster, the corfu server port is different(9000, 9002, 9004...) and they must be explicitly set by providing a value to this param.  So if the check is eliminated, a value of '0' can be set, which is unintentional, right?

Reply:
I see, this is an intermediate object to actually build the command from different constructors used in the tests. Still, there is sort of a miss, 'Corfu-Port' is required by LR to startup, so even if this value is '0' it means we would not be able to connect  to a Corfu Server and nothing should work.